### ISSUE TITLE
start the platon node faild

### ISSUE THREAD

**Issue by benbaley** [PlatONnetwork/PlatON-Go#1479]
Hi there,

please note that this is an issue tracker reserved for bug reports and feature requests.

#### System information

PlatON version: 0.13.1
OS & Version: Windows/Linux/OSX
Commit hash :4f6b0ce5f29fef7fb43402ed44d5fafe2dd5d237

#### Expected behaviour
start OK

#### Actual behaviour
failed

#### Steps to reproduce the behaviour

platon --identity platon --datadir ./data --port 16701 --rpcport 6711 --rpcapi "db,debug,platon,net,web3,admin,personal,txpool,miner" --rpc --nodekey

#### Backtrace

````
INFO [08-10|17:12:07.662|core/snapshotdb/snapshotdb.go:226]  open snapshot db Allocated cache and file handles package=snapshotdb cache=768      handles=524288 baseDB=true
DEBUG[08-10|17:12:07.699|eth/backend.go:142]                 read header number from chain db         height=<nil>
INFO [08-10|17:12:07.699|core/genesis.go:156]                Writing default main-net genesis block 
WARN [08-10|17:12:07.699|common/address.go:93]               the address prefix not set yet           input=lat1dl93r6fr022ca5yjqe6cgkg06er9pyqfwkwwdg
panic: invalid hex string

goroutine 1 [running]:
github.com/PlatONnetwork/PlatON-Go/common/hexutil.MustDecode(...)
        /home/jht/go/src/github.com/PlatONnetwork/PlatON-Go/common/hexutil/hexutil.go:80
github.com/PlatONnetwork/PlatON-Go/core.DefaultGenesisBlock(0x21be180)
        /home/jht/go/src/github.com/PlatONnetwork/PlatON-Go/core/genesis.go:498 +0x521
github.com/PlatONnetwork/PlatON-Go/core.SetupGenesisBlock(0x7f8b0606cd80, 0xc000266000, 0x7f8b0008f7a0, 0xc0005fa0a0, 0x0, 0x2, 0x0, 0x0, 0x0, 0x0, ...)
        /home/jht/go/src/github.com/PlatONnetwork/PlatON-Go/core/genesis.go:157 +0x850
github.com/PlatONnetwork/PlatON-Go/eth.New(0xc0000e2a00, 0xc0000bc700, 0x30, 0x13da5e0, 0xc0004fb7c8)
        /home/jht/go/src/github.com/PlatONnetwork/PlatON-Go/eth/backend.go:200 +0x3e3
github.com/PlatONnetwork/PlatON-Go/cmd/utils.RegisterEthService.func2(0xc0000e2a00, 0xc0002ea450, 0xc0004fb980, 0xc0002a8460, 0x2)
        /home/jht/go/src/github.com/PlatONnetwork/PlatON-Go/cmd/utils/flags.go:1280 +0x46
github.com/PlatONnetwork/PlatON-Go/node.(*Node).Start(0xc00012a800, 0x0, 0x0)
        /home/jht/go/src/github.com/PlatONnetwork/PlatON-Go/node/node.go:200 +0x414
github.com/PlatONnetwork/PlatON-Go/cmd/utils.StartNode(0xc00012a800)
        /home/jht/go/src/github.com/PlatONnetwork/PlatON-Go/cmd/utils/cmd.go:67 +0x2f
main.startNode(0xc000032580, 0xc00012a800)
        /home/jht/go/src/github.com/PlatONnetwork/PlatON-Go/cmd/platon/main.go:310 +0x7e
main.platon(0xc000032580, 0xc000032580, 0xc0004fbb9f)
        /home/jht/go/src/github.com/PlatONnetwork/PlatON-Go/cmd/platon/main.go:298 +0xe3
gopkg.in/urfave/cli%2ev1.HandleAction(0x1259540, 0x14aec90, 0xc000032580, 0xc0004ba480, 0x0)
        /home/jht/go/pkg/mod/gopkg.in/urfave/cli.v1@v1.20.0/app.go:490 +0xc8
gopkg.in/urfave/cli%2ev1.(*App).Run(0xc0003be000, 0xc0000d4000, 0x16, 0x16, 0x0, 0x0)
        /home/jht/go/pkg/mod/gopkg.in/urfave/cli.v1@v1.20.0/app.go:264 +0x58c
main.main()
        /home/jht/go/src/github.com/PlatONnetwork/PlatON-Go/cmd/platon/main.go:284 +0x55
````

**Comment by benbaley** [comment]
Fixed in #1480

**Comment by benbaley** [comment]
close by done
